application-task-manager-default/src/test/java/com/xwiki/taskmanager/IntegrationTests.java

```diff
@@ -48,9 +54,9 @@
  */
 @RunWith(RenderingTestSuite.class)
 @AllComponents
+@RenderingTestSuite.Scope(value = ""/*, pattern = "task1.test"*/)
```

Comment:
If the commented code is not needed (it's not documentation) then you should remove it.